ci/check_generated_docstrings.sh

```diff
@@ -6,19 +6,15 @@
 set -euo pipefail
 
 VENV=build/venv-docstrings
-CLANG_FORMAT_VERSION=9.0.0
+CLANG_VERSION=6.0.0.2
```

Comment:
:cold_face:

Reply:
pybind11 actually has the required clang bindings as a submodule, so I've removed this